storybook/storybook-react/src/Header/Header.stories.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Header, PageMenu } from '@amsterdam/design-system-react'
+import { LoginIcon, MenuIcon } from '@amsterdam/design-system-react-icons'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Containers/Header',
+  component: Header,
+} satisfies Meta<typeof Header>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {}
+
+export const WithSiteTitle: Story = {
+  args: {
+    title: 'Aan de Amsterdamse Grachten',
+  },
+}
+
+export const WithLogoVariant: Story = {
+  args: {
+    logoBrand: 'ggd-amsterdam',
+    logoLink: 'https://www.ggd.amsterdam.nl/',
+    logoLinkTitle: 'Naar de website van de GGD Amsterdam',
```

Comment:
I think we've discussed that we do need to add the site title here for screen readers because the logo is "silent" and they do not know on what site they are.